webapp/src/main/resources/sass/mojito.scss

```diff
@@ -1017,3 +1017,159 @@ label.current-pageNumber {
         "newUser .       ."
     ;
 }
+
+.jobs-container {
+    padding-top: 10px;
+    text-align: center;
+}
+
+.job-row {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+    margin: 0 auto;
+    border: 1px solid rgba(128, 128, 128, 0.25);
+    min-width: 450px;
+    max-width: 700px;
+    color: gray;
+    padding: 1.5em;
```

Comment:
It's generally better to use `rem` for this.
In general `rem` is better than `px` too but our app uses `px` so that's fine.